src/InBrain.tsx

```diff
@@ -157,6 +173,8 @@ const optionsActions: { [key in StylingOptionName]: ((params: any) => Promise<an
 export default {
     init,
     showSurveys,
+    setSessionID,
+    setDataOptions,
```

Comment:
Remove setDataOptions from here if possible (I'm not sure what is that, if this is just list of available methods - no problem, let it be then)

Reply:
yes, it's listing of available methods